### ISSUE TITLE
http://cssnano.co/usage/ is not available

### ISSUE THREAD

**Issue by stereokai** [ben-eb/cssnano#418]
"Installation guide for your build process." link is missing.

**Comment by blieque** [comment]
I'm still getting a 404 for `/usage/`. Similarly, I can't load `/options/`, or find any other API reference documentation for cssnano.

See ben-eb/gulp-cssnano#95.

**Comment by andyjansson** [comment]
Yes, when we rewrote the website for the v4 release, we also implemented a new URL structure. The README.md for the 3.x release linked to the old website, but v4 has now been released so it shouldn't be an issue.

This is how you configure the presets: https://cssnano.co/guides/presets#options-syntax
This is what optimisations are available: https://cssnano.co/guides/optimisations

**Comment by andyjansson** [comment]
I agree, there are many improvements to be done to the website and I've been planning to address them at some point. Unfortunately, we have more pressing issues which take priority.

**Comment by blieque** [comment]
Once I get webpack + OptimizeCSSAssetsPlugin + cssnano + PostCSS + autoprefixer all working how they should, I'd be happy to write some kind of guide if it would save some others some time. From what I gather, this setup is pretty defacto for webpack bundling. Is there a good place this could go?
